media-gfx/blender/blender-9999.ebuild

```diff
@@ -28,9 +28,9 @@ LICENSE="|| ( GPL-3 BL )"
 IUSE="+bullet +dds +fluid +openexr +tbb \
```

Comment:
Backslash escapes are meaningless here, it's fine to have multiline string as IUSE.